app/views/helpers/calcBreakdownHelper.scala.html

```diff
@@ -15,23 +15,39 @@
 
 @(calcModel: CalcDisplayModel, taxYear: Int)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig, user: auth.MtdItUser[_], sources: models.IncomeSourcesModel)
 
 @defining(calcModel.calcDataModel.get) { breakdown =>
     <table class="itvc-table">
         <tbody>
-        @if((sources.hasBusinessIncome && sources.businessIncomeSources.exists(_.accountingPeriod.determineTaxYear == taxYear)) || breakdown.profitFromSelfEmployment > 0){
+
         <tr id="business-profit-section" class="no-border-bottom">
-            <td id="business-profit-heading">@messages("estimated_tax_liability.calc-breakdown.business-profit")</td>
-            <td id="business-profit"  class="summary-data">@{breakdown.profitFromSelfEmployment.toCurrency}</td>
-        </tr>
-        }
-        @if((sources.hasPropertyIncome && sources.propertyIncomeSource.get.accountingPeriod.determineTaxYear == taxYear) || breakdown.profitFromUkLandAndProperty > 0){
-        <tr id="property-profit-section" class="no-border-bottom">
-            <td id="property-profit-heading">@messages("estimated_tax_liability.calc-breakdown.property-profit")</td>
-            <td id="property-profit" class="summary-data">@{breakdown.profitFromUkLandAndProperty.toCurrency}</td>
+            <td id="business-profit-heading">
+                @{
+                    if((sources.hasBusinessIncome && sources.businessIncomeSources.exists(_.accountingPeriod.determineTaxYear == taxYear)) || breakdown.profitFromSelfEmployment > 0) {
+                        messages("estimated_tax_liability.calc-breakdown.business-profit")
+                    } else if((sources.hasPropertyIncome && sources.propertyIncomeSource.get.accountingPeriod.determineTaxYear == taxYear) || breakdown.profitFromUkLandAndProperty > 0) {
+                        messages("estimated_tax_liability.calc-breakdown.property-profit")
+                    }
+                }
+                @if(breakdown.bbsiIncome > 0 && calcModel.calcStatus == Estimate) {@messages("estimated_tax_liability.calc-breakdown.and-savings")}
+            </td>
+
+            <td id="business-profit"  class="summary-data">
+                @{
+                    if(calcModel.calcStatus == Estimate){(breakdown.profitFromSelfEmployment + breakdown.profitFromUkLandAndProperty + breakdown.bbsiIncome).toCurrency}
+                    else { (breakdown.profitFromSelfEmployment + breakdown.profitFromUkLandAndProperty).toCurrency}
+                }
+            </td>
         </tr>
+
+        @if(calcModel.calcStatus == Crystallised && breakdown.bbsiIncome > 0){
+            <tr id="savings-income-section"  class="no-border-bottom">
+                <td id="savings-income-heading">@messages("estimated_tax_liability.calc-breakdown.income-savings")</td>
+                <td id="savings-income" class="summary-data">@breakdown.bbsiIncome.toCurrency</td>
+            </tr>
         }
 
         <tr id="personal-allowance-section">
```

Comment:
**Estimates Page**
Personal Allowance Section needs to be updated to include a message which says `, incl. Savings (for period reported)`

As per the prototype here:
https://www.prototypes.tax.service.gov.uk/itvc-prototype/2-7-0/income-tax-estimate?taxYear=2019

**Bills Page**
Personal Allowance Section needs to be updated to include a message which says `(Income Tax and savings)`

As per the prototype here:
https://www.prototypes.tax.service.gov.uk/itvc-prototype/2-7-0/crystalisation